includes/templates/class-amp-post-template.php

```diff
@@ -369,13 +367,18 @@ private function build_post_featured_image() {
 			]
 		);
 
+		$featured_image = get_post( $featured_id );
+		if ( ! $featured_image ) {
+			return;
+		}
```

Comment:
Shouldn't this condition to short-circuit rather be moved above? No need to construct `$dom` or `$assets` if it's just going to `return`.

Reply:
Fair enough https://github.com/ampproject/amp-wp/pull/3360/commits/e30638fa8fb6b576d6c812234c50dd03d3034581